routes/v1.js

```diff
@@ -305,6 +429,11 @@ api.declare({
     return;
   }
 
+  // Ensure group membership is declared, and that schedulerId isn't conflicting
+  if (!await ensureTaskGroup(this, taskId, taskDef, res)) {
+    return;
```

Comment:
`res` is passed to `ensureTaskGroup` which will send an error when it returns `false`..

It's a bit of a hack, but I avoid typing the same error message twice...